### PR TITLE
Akron Release

### PR DESCRIPTION
**Pull request by greer184**
This pull request introduces the initial implementation of the Open Pond Protocol as well as providing a basic API to begin writing applications as well as an example application (codename: Akron) to trial the protocol.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +16,80 @@ Once the environment has been configured, compile and run the binary, entering t
...
-// Command to run (Example)
-cargo run "127.0.0.1:8091" "127.0.0.1.8090"
+// Command to run (Example Node)
+cargo run config/example.toml
+```
+
+This command will start a node and if connected to another node will enable communication between
+the two nodes. In order for anything useful to be communicated, each node needs to have 1 or more
+applications connected to their node. The applications must have matching IDs to communicate. These
+IDs can be configured in the node configuration file described later in this document. To run the
```
Comment (greer184) [inline README.md]: nit: can be configured -> can be set

File: examples/akron/main.rs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    // Spawn threads to send requests, service requests, and read responses
+    thread::spawn(|| request_updates(request_endpoint, name_ref));
+    thread::spawn(|| service(service_endpoint, last_message_ref));
+    thread::spawn(|| receive_updates(response_endpoint));
+
+    // Store messages from standard input
+    loop {
+        thread::sleep(time::Duration::new(1, 0));
+        let mut input = String::new();
+        if io::stdin().read_line(&mut input).is_ok() {
```
Comment (greer184) [inline examples/akron/main.rs:36]: if read line is blocking then probably do not need the sleep call

**Review by nhanby46 — APPROVED** [review]
All looks good!

File: examples/akron/main.rs (hunk 2 above)
Comment (greer184) [reply examples/akron/main.rs:36]: good guess, it's blocking

**Review by nhanby46 — APPROVED** [review]

**Pull request merged**